loader/loader_test.go

```diff
@@ -264,7 +264,7 @@ var _ = Describe("Loader", Ordered, func() {
 				DisplayName:        "my-environment",
 				DeploymentStrategy: applicationapiv1alpha1.DeploymentStrategy_Manual,
 				ParentEnvironment:  "",
-				Tags:               []string{"ephemeral"},
+				Tags:               []string{},
```

Comment:
I think we should remove the `hasEnv` object entirely since `Environment` CRDs have been removed from application-api.